test/unit/algorithm_mode/test_custom_metrics.py

```diff
@@ -47,12 +48,6 @@ def test_binary_f1_logistic():
     assert f1_score_result == 1/3
 
 
-def test_mse():
```

Comment:
Curious why we remove mse in test?

Reply:
mse should not be applied to classification problems.